scRNA-seq-advanced/04-differential_expression.Rmd

```diff
@@ -171,32 +275,26 @@ library_ids <- c(
   "SCPCL000491"
 )
 
-# subset sce to only contain samples with library IDs of interest 
+# subset sce to only contain samples with IDs of interest 
 samples_to_keep <- integrated_sce$sample %in% library_ids
 rms_sce <- integrated_sce[,samples_to_keep]
 
-# remove original integrated object 
+# remove original integrated object to save some memory
 rm(integrated_sce)
-```
 
-```{r}
-# we are left with 3 from each subdiagnosis 
-sample_metadata %>%
-  dplyr::filter(library_id %in% library_ids) %>%
-  dplyr::count(subdiagnosis)
+# print out our new sce 
+rms_sce
 ```
 
 
-```{r}
-# look at distribution of cell types across all libraries 
-as.data.frame(colData(rms_sce)) %>%
-  dplyr::count(celltype_broad) %>%
-  dplyr::arrange(desc(n))
-```
-
-The `Tumor_Myoblast` is the most prominent cell type, and based on the UMAPs is found in all 6 of our samples.
-We will start by comparing the gene expression of the myoblasts across ARMS and ERMS samples.
+We now have an updated SCE object that contains 6 samples that were obtained from a mix of ARMS and ERMS patients.
+We are particularly interested in identifying DE genes in tumor cell types between ARMS and ERMS.
```

Comment:
I'd spell out DE here since it's got some grammar vs DE which I tend to read as "differential expression" (but is that just me?)
```suggestion
We are particularly interested in identifying differentially expressed genes for specific tumor cell types between ARMS and ERMS.
```

Reply:
I'll note one could phrase this as a question, something like: "We want to ask, Are specific tumor cell types differentially expressed between ARMS and ERMS?"

Ok either way!